includes/collection/class-actors.php

```diff
@@ -389,4 +389,255 @@ public static function get_type_by_id( $user_id ) {
 
 		return 'user';
 	}
+
+	/**
+	 * Create or update a remote Actor (e.g., a follower) as a custom post type.
+	 *
+	 * @param array|Actor $actor_data The ActivityPub actor object as associative array (must include 'id').
+	 *
+	 * @return int|\WP_Error The post ID or WP_Error.
+	 */
+	public static function add( $actor_data ) {
```

Comment:
I wonder if it would be clearer to split this function into `create()` (or `insert()`) and `update()`? 

`add()`, to me, would be used more in the context of adding to something already existing, like [`WP_Error::add()`](https://developer.wordpress.org/reference/classes/wp_error/add/).